meow-tutor.el

```diff
@@ -108,43 +107,42 @@
 
  Note: If you want to move the cursor while in Insert mode,
        you may use the arrow keys instead of exiting and
-       reentering Insert mode.
+       re-entering Insert mode.
  Note: The modeline will display your current mode.
-       Notice that when you press \\[meow-insert], 'NORMAL' changes to 'INSERT'.
```

Comment:
The default state indicator for meow is:
```
((normal . "NORMAL")
 (motion . "MOTION")
 (keypad . "KEYPAD")
 (insert . "INSERT")
 (beacon . "BEACON"))
```
We could get the value from the editor itself:
```
"%s %s"
(alist-get 'normal meow-replace-state-name-list) 
(alist-get 'insert meow-replace-state-name-list)
```

What do you think?

This PR improves the quality greatly! Thanks!

Reply:
Sounds good! My elisp knowledge is not great, I don't think I can do this properly. Can you please do it after I merge the PR?

Reply:
No problem! I’m also not familiar with elisp, just learn by doing 😸

Reply:
ya, we should just use the same name in document. the indicator is just a display. 
we don't have to use it in tutor.